style.css

```diff
@@ -0,0 +1,674 @@
+    display: flex;
+    flex-direction: column;
+}
+
+/* text-card */
+.text-card-demo {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-direction: row;
+    width: 100%;
+    border: 0.2rem solid lightgray;
+}
+.text-card {
+    padding: 1rem;
+    width: 200px;
+    height: auto;
+    margin: 1rem;
+    border-radius: 1.25rem;
+    background-color: #fff;
+    font-size: 1.2rem;
+}
+
+/* yt-card */
+.yt-card {
+    width: 250px;
+    height: 300px;
+    padding: 1rem;
+}
+.yt-card-img img {
+    width: 100%;
+}
+.yt-left img {
+    width: 4rem;
+    border-radius: 50%;
+    margin: 1rem;
+}
+.yt-right p {
+    font-size: 1.3rem;
+}
+.yt-card-body {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+}
+
+/* lists */
+.main-list {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.list-body {
+    width: 200px;
+    /* height: 350px; */
+    padding: 1rem;
+    background-color:white;
+    border-radius: 2rem;
+    list-style: none;
+    margin: 1rem;
+}
+.list {
+    padding: 1rem 2rem;
+    background-color: #333;
+    color: white;
+    margin: 1rem;
+    font-size: 2rem;
+    text-align: center;
+    border-radius: 1rem;
+}
+.navigation-body {
+    display: flex;
+    flex-direction: row;
+    align-items:center;
+    justify-content: space-between;
+    width: 100%;
+    background-color: var(--primary-color);
+    padding: 1rem;
+}
+.nav-ul {
+    list-style: none;
+}
+.nav-li {
+    padding: 0.5rem 1rem;
+    background-color: #333;
+    color: white;
+    margin-right: 0.5rem;
+    font-size: 1.25rem;
+    text-align: center;
+    border-radius: 1rem;
+}
+.nav-li a {
+    text-decoration: none;
+    color: #fff;
+}
+.nav-right-ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+
+}
+.nav-left-ul {
+    padding-left: 0;
+}
+
+/* forms */
+.main-form {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    width: 100%;
+    align-items: center;
+}
+.form {
+    width: 300px;
+    height: auto;
+    padding: 1rem;
+    border: 1px solid black;
+    border-radius: 2rem;
+    margin: 1rem;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: space-evenly;
+}
+.form label {
+    font-size: 1.25rem;
+    font-weight: bold;
+}
+.form input {
+    border: none;
+    border-radius: 1rem;
+    padding: 1rem;
+    margin: 0.5rem;
+    border: 0.1rem solid lightgray;
+}
+
+/* Media Queries */
+@media (max-width : 889px) {
+/* body {
+    background-color: red;
+}
+.container {
+    background-color: red;
+} */
+.header-text {
+    font-size: 4rem;
+}
+.header-content {
+    padding: 1rem;
+}
+.alert-side {
+    flex-direction: column;
+    height: 60vh;
+}
+.main-card {
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+}
+.text-card-demo {
+    flex-direction: column;
+}
+}
+
```

Comment:
Way too many media queries ! Try to optimize the components for just two or three sizes `480px` , `768px` and `1200px`, you shouldn't need to add more breakpoints than these.

Reply:
Yeah, I have added because, I have been debugging in the responsive mode. I will try to minimise the no.of media queries as you suggested.